wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -401,6 +404,27 @@ public RepeatCommand repeatedly() {
     return new RepeatCommand(this);
   }
 
+  /**
+   * Decorates this command to run repeatedly, restarting until the command runs for the given times
+   * amount. The decorated command can still be canceled.
```

Comment:
```suggestion
   * Decorates this command to run repeatedly, restarting until the command runs for the given number
   * of times. The decorated command can still be canceled.
```
Reads a lot more naturally.

Reply:
Added in with `Better comments and variable names` commit